compiler/qsc_frontend/src/typeck/tests.rs

```diff
@@ -283,17 +282,17 @@ fn call_generic_length() {
             }
         "},
         "Length([true, false, true])",
-        &expect![[r#"
-            #7 58-69 "(xs : 'T[])" : ?
-            #8 59-68 "xs : 'T[]" : ?
-            #17 98-125 "Length([true, false, true])" : Int
-            #18 98-104 "Length" : (Bool[] -> Int)
-            #21 104-125 "([true, false, true])" : Bool[]
-            #22 105-124 "[true, false, true]" : Bool[]
-            #23 106-110 "true" : Bool
-            #24 112-117 "false" : Bool
-            #25 119-123 "true" : Bool
-        "#]],
+        &expect![[r##"
```

Comment:
nit: The expects still occasionally put the double `##` in when they aren't needed. Please remove the extra `#` on the expect of these tests.